integration-cli/docker_cli_search_test.go

```diff
@@ -11,83 +11,50 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 	testRequires(c, Network, DaemonIsLinux)
 
 	out, exitCode := dockerCmd(c, "search", "busybox")
-	if exitCode != 0 {
-		c.Fatalf("failed to search on the central registry: %s", out)
-	}
+	c.Assert(exitCode, check.Equals, 0, check.Commentf("failed to search on the central registry: %s", out))
 
-	if !strings.Contains(out, "Busybox base image.") {
-		c.Fatal("couldn't find any repository named (or containing) 'Busybox base image.'")
-	}
+	c.Assert(strings.Contains(out, "Busybox base image."), check.Equals, true, check.Commentf("couldn't find any repository named (or containing) 'Busybox base image.'"))
 }
 
 func (s *DockerSuite) TestSearchStarsOptionWithWrongParameter(c *check.C) {
 	out, exitCode, err := dockerCmdWithError("search", "--stars=a", "busybox")
-	if err == nil || exitCode == 0 {
-		c.Fatalf("Should not get right information: %s, %v", out, err)
-	}
-
-	if !strings.Contains(out, "invalid value") {
-		c.Fatal("couldn't find the invalid value warning")
-	}
+	c.Assert(err, check.Not(check.IsNil), check.Commentf("Should not get right information: %s, %v", out, err))
```

Comment:
There are two ways to do this, one is `NotNil` and the other is `Not(IsNil)`, for consistent with others like check.Not(check.Equauls), I choose the `check.Not(...)`